examples/pipelining/nyu_rest_preproc.ini

```diff
@@ -30,12 +30,12 @@
 
 # Path (relative or full) of directory containing data (if different from directory
 # containing this configuration file).
-dataset_dir = 
+dataset_dir =
```

Comment:
Something wrong here

Reply:
I've removed the NYU rest ini file. The NYU rest example is obviously not the only one showcasing use of ini files. Was there a specific reason for including this dataset?

Reply:
O that's an ini file, sorry.
Anyway, if the fetcher is deprecated, it's better to remove it (as you may have seen, Nilearn has examples on a new resting state dataset).
Best,
B